README.md

```diff
@@ -1,16 +1,20 @@
-# 💊 ReVanced Manager
+<p align="center">
+  <img width="800" src="Logo.png">
```

Comment:
This doesn't follow any style guide. Please refer to [this](https://cdn.discordapp.com/attachments/952987428786941952/1067552057940267079/revanced-brand-identity.zip).

Reply:
What is this 😵‍💫😵‍💫

Btw OsuMatrix, Circular logo where?

Reply:
Masked off the full logo in the zip.

Reply:
This is not even the logo i updated 🙁
It still the old one

Reply:
I can't find the updated one, also it's not open source, might want to create a repository for it.

Reply:
[Here](https://cdn.discordapp.com/attachments/953965039105232906/1070354383159709798/revanced-brand-identity.zip)

Reply:
Sorry for the inconvenience. I chose the logo from the `res` folder in the main code. I will make sure to abide by the official logos

Reply:
> Sorry for the inconvenience. I chose the logo from the `res` folder in the main code. I will make sure to abide by the official logos.

Logo & typo both

Reply:
@oSumAtrIX what about this wordmark?
![revanced-manager-wordmark](https://user-images.githubusercontent.com/82371061/218303047-45f601be-9e67-4cf5-ad2d-0fed47c4bbbd.svg)